praisonai_tools/video/motion_graphics/backend_html.py

```
@@ -0,0 +1,320 @@
+            str(output_path)
+        ]
+        
+        # Run FFmpeg
+        process = await asyncio.create_subprocess_exec(
+            *cmd,
+            stdout=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.PIPE
+        )
+        
+        try:
+            stdout, stderr = await asyncio.wait_for(
```

Ruff (0.15.10)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)